pyface/tasks/i_editor_area_pane.py

```diff
@@ -42,22 +42,22 @@ class IEditorAreaPane(ITaskPane):
 
     # 'IEditorAreaPane' interface -----------------------------------------#
 
-    # The currently active editor.
+    #: The currently active editor.
     active_editor = Instance(IEditor)
 
-    # The list of all the visible editors in the pane.
+    #: The list of all the visible editors in the pane.
     editors = List(IEditor)
 
-    # A list of extensions for file types to accept via drag and drop.
-    # Note: This functionality is provided because it is very common, but drag
-    # and drop support is in general highly toolkit-specific. If more
-    # sophisticated support is required, subclass an editor area implementation.
+    #: A list of extensions for file types to accept via drag and drop.
+    #: Note: This functionality is provided because it is very common, but drag
+    #: and drop support is in general highly toolkit-specific. If more
+    #: sophisticated support is required, subclass an editor area implementation.
```

Comment:
(Feel free to ignore, but I wonder if this makes the line too long for the nonexisting flake8...)

Reply:
I confirmed it does trigger flake8 (some other changes in this PR do as well), but there are many other flake8 errors in the files in this PR not related to this PR.